templates/nextjs/template/app/layout.tsx

```diff
@@ -1,9 +1,17 @@
 import type { Metadata, Viewport } from 'next';
+import { Inter } from 'next/font/google';
 
 import '@sumup-oss/design-tokens/light.css';
 import '@sumup-oss/circuit-ui/styles.css';
 
-import { PreloadResources } from './preload-resources';
+const inter = Inter({
+  subsets: ['latin'],
+  // FIXME: Re-enable once https://github.com/vercel/next.js/issues/68395 has been resolved
```

Comment:
Should this phrasing be for the template developer or the developer user? 🤔 
Or maybe mention to the later, "You can uncomment this line it if you have Next.js > 14.x.y" ?

Reply:
It's a note to myself (the template developer). The PR has already been merged; I'm just waiting for a new version to be released.